web-marketplace/src/hooks/projects/useProjectsWithOrders.ts

```diff
@@ -102,7 +110,8 @@ export function useProjectsWithOrders({
 
   const { data: projectsData, isFetching: isLoadingProjects } = useQuery(
     getProjectsQuery({
-      enabled: !classId && !projectId && !!ecocreditClient,
+      enabled:
+        !isOffChainProject && !classId && !projectId && !!ecocreditClient,
```

Comment:
unrelated to this task but just a small optimization I spotted (in case of using this on an offchain project page, we were running `getProjectsQuery` because in this case `projectId` is not defined so enabled was true